src/Vocalizer/Vocalizer.cpp

```diff
@@ -26,6 +40,36 @@ int main(array<System::String^>^ args)
     SpeechSynthesizer speaker;
     speaker.Rate = 1;
     speaker.Volume = 100;
+
+    // The Vocalizer component is intended to sonically convey text to a human
+    // operator, and the Automation Voice is inappropriate for this purpose. In
+    // addition, the use of the Automation Voice in this context could trigger
+    // non-recoverable recursion.
+    //
+    // By default, the SpeechSynthesizer instance declared above will use the
+    // system's default voice. Explicitly set the instance's voice to any
+    // available value other than the Automation Voice in order to avoid
+    // problems in cases where the system has designated the Automation Voice
+    // as the "default" voice.
+    if (isAutomation(speaker.Voice->Name))
+    {
+        for each (InstalledVoice^ voice in speaker.GetInstalledVoices())
+        {
+            if (!isAutomation(voice->VoiceInfo->Name))
+            {
+                speaker.SelectVoice(voice->VoiceInfo->Name);
+                break;
+            }
+        }
+        if (isAutomation(speaker.Voice->Name))
+        {
+            Console::WriteLine(gcnew System::String(
```

Comment:
`/clr` flag is set on Vocalizer so you can do `Console::WriteLine("Unable to locate an authentic voice.")`.

Reply:
Good call! I'll take care of this in a follow up since address it consistently will involve changing code unrelated to the intent of this patch.